src/Routes.js

```diff
@@ -51,6 +53,15 @@ export const Routes = () => {
                     }
                     rootClass="inventory"
                 />
+                <Route exact path={routes.groupDetail} component={groupsEnabled
+                    ? InventoryGroupDetail
+                    : () => (
+                        <EmptyState>
```

Comment:
Is not this component exist on FEC with an already empty state? If this is the component for 404, it should be there, ready for consumption. Also, maybe we can extract this into some wrapper and avoid repetition?

Reply:
Didn't found the empty state-like components that could replace it. if you could point me out to any would be cool. meanwhile, I extracted this empty state to a separate component in the last commit 👍🏼